README.rst

```diff
@@ -62,65 +176,230 @@ A simple example to get you up and running:
+| Circuits with thousands of qubits and millions  | * |stim|_            |
+| of Clifford operations?                         |                      |
++-------------------------------------------------+----------------------+
+| Quantum error correction (QEC)?                 | * |stim|_            |
++-------------------------------------------------+----------------------+
+| Chemistry and/or material science?              | * |openfermion|_     |
+|                                                 | * |openfermioncirq|_ |
++-------------------------------------------------+----------------------+
+| Quantum machine learning (QML)?                 | * |tfq|_             |
++-------------------------------------------------+----------------------+
+| Real experiments using Cirq?                    | * |recirq|_          |
++-------------------------------------------------+----------------------+
+
+.. ▶︎─── start github-only ───
+.. raw:: html
+
+   </div>
+.. ▶︎─── end github-only ───
+
+
+Community
+---------
+
+Cirq has benefited from open-source contributions by over 200 people and
+counting. We are dedicated to cultivating an open and inclusive community to
+build software for quantum computers, and have a `code of conduct
+<https://github.com/quantumlib/cirq/blob/main/CODE_OF_CONDUCT.md>`__ for our
+community.
+
+
+Announcements
+.............
+
+Stay on top of Cirq developments using the approach that best suits your needs:
+
+* For releases and major announcements: sign up to the low-volume mailing list
+  `cirq-announce <https://groups.google.com/forum/#!forum/cirq-announce>`__.
+
+* For releases only:
+
+  * Via GitHub notifications: configure `GitHub repository notifications
+    <https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository>`__
+    for the Cirq repo.
+
+  * Via Atom/RSS from GitHub: subscribe to the GitHub `Cirq releases Atom feed
+    <https://github.com/quantumlib/Cirq/releases.atom>`__.
+
+  * Via RSS from PyPI: subscribe to the PyPI `Cirq releases RSS feed
+    <https://pypi.org/rss/project/cirq/releases.xml>`__.
+
+Cirq releases take place approximately every quarter.
+
+
+Questions and Discussions
+.........................
+
+.. |cirq| replace:: ``cirq``
+.. _cirq: https://quantumcomputing.stackexchange.com/questions/tagged/cirq
+
+* Do you have questions about using Cirq? Post them to the `Quantum Computing
+  Stack Exchange <https://quantumcomputing.stackexchange.com/>`__ and tag them
+  with the |cirq|_ tag.
+
+* Would you like to get more involved? *Cirq Cynq* is our biweekly virtual
+  meeting of contributors to discuss everything from issues to ongoing
+  efforts, as well as to ask questions. Join the `cirq-dev Google Group
+  <https://groups.google.com/forum/#!forum/cirq-dev>`__ to get an automatic
+  meeting invitation.
```

Comment:
Maybe a quick note that this is for group membership and we don't typically post messages there, in case people see no one has posted in forever.

Reply:
Oh yes, that's a good point. I'll add a note.

Reply:
Alright, I reworded this a little bit – how is it now?